src/c++/library/CMakeLists.txt

```diff
@@ -400,6 +501,7 @@ if(TRITON_ENABLE_CC_HTTP OR TRITON_ENABLE_CC_GRPC OR TRITON_ENABLE_PERF_ANALYZER
       FILES
       ${CMAKE_CURRENT_SOURCE_DIR}/common.h
       ${CMAKE_CURRENT_SOURCE_DIR}/ipc.h
+      ${CMAKE_CURRENT_SOURCE_DIR}/cencode.h
```

Comment:
Did you time the initial build and iterative builds (with no changes) before/after these changes (same for the changes in the PA repo)? Just to make sure we're not extending the build time unintentionally. I don't think your changes will do this, but if we have not timed before/after, it would be helpful (they may even reduce the build time!).

Reply:
This is a good point. We talked about taking this measurement earlier on.

Build times enabling all optional features and using 8 cores:

**Original Client Repo Build**
**Initial:** 9m45s
**Rebuild:** 5m13s

**PA Repo Build**
**Initial:** 10m6s
**Rebuild**: 5m22s

There seems to be a slight discrepancy between the two, with the PA repo build being a little slower. I'm not sure how much of this can be chalked up to including the client repo as another ExtneralProject_Add. It warrants further investigation, but I don't see this as a blocker for this PR, especially given we plan on optimizing our CMake builds soon. Open to thoughts though.

Reply:
Thanks for timing and sharing the results. The times do not differ significantly. Resolving this thread.